reconcile/quay_permissions.py

```diff
@@ -66,17 +84,28 @@ def run(dry_run):
                             logging.warning('wrong service kind, ' +
                                             'should be quay-membership')
                             continue
-                        if permission['org'] != org_name:
+
+                        perm_org_key = (
+                          permission['quayOrg']['instance']['name'],
+                          permission['quayOrg']['name']
+                        )
+
+                        if perm_org_key != org_key:
                             logging.warning('wrong org, ' +
```

Comment:
You don't need a `+` here:
```
>>> i=42
>>> 'Life, the universe and everything else are ' \
... f'{i}'
'Life, the universe and everything else are 42'
```

Reply:
fwiw this predates this PR, however I still haven't got a hang on multiline strings, sometimes I enclose them in parenthesis so it allows me to split them into multiple lines. I also dislike the `+` because it creates more objects in memory, vs the interpreter creating a single string